packages/website/src/css/prism.css

```diff
@@ -6,16 +6,16 @@
   --token-color-doctype: #008000;
   --token-color-string: #a31515;
   --token-color-operator: #393a34;
-  --token-color-symbol: #36acaa;
+  --token-color-symbol: #277c7b;
   --token-color-number: #098658;
   --token-color-keyword: #00f;
-  --token-color-function: #569cd6;
+  --token-color-function: #2b74b1;
   --token-color-function-variable: #000;
   --token-color-important: #e90;
-  --token-color-class-name: #2b91af;
+  --token-color-class-name: #237690;
   --token-color-selector: #800000;
   --token-color-regexp: #800000;
-  --token-color-property: #d00;
+  --token-color-property: #c70000;
```

Comment:
[Curious]

How did you choose those specific colors?

Reply:
I ran axe devtools to see which colors on each page were below the 4.5:1 threshold and then I used [this website](https://webaim.org/resources/contrastchecker/) to check the contrast ratio and modify the current foreground color until it was over that amount. Then I re-ran the devtools to verify the changes were adequate 🙂.

Reply:
Big +1 on https://webaim.org/resources/contrastchecker, great tool!